package.json

```diff
@@ -38,5 +38,9 @@
     "format": "prettier --write '**/*.{md,css,js,json,yaml,yml}'",
     "build": "node build.js",
     "test:ci": "npm run lint -s"
+  },
+  "dependencies": {
+    "file-system": "^2.2.2",
+    "prismjs": "^1.15.0"
```

Comment:
As far as I can see Prism is loaded from the file in public, not the one installed here. Could we remove this?